src/main/java/de/muenchen/isi/infrastructure/entity/AbfragevarianteSachbearbeitung.java

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright (c): it@M - Dienstleister für Informations- und Telekommunikationstechnik
+ * der Landeshauptstadt München, 2022
+ */
+package de.muenchen.isi.infrastructure.entity;
+
+import de.muenchen.isi.infrastructure.entity.enums.lookup.SobonOrientierungswertJahr;
+import java.math.BigDecimal;
+import java.util.List;
+import javax.persistence.*;
+import lombok.Data;
+import lombok.ToString;
+
+@Embeddable
+@Data
+@ToString(callSuper = true)
+public class AbfragevarianteSachbearbeitung {
+
+    @Column(precision = 10, scale = 2, nullable = true)
+    private BigDecimal geschossflaecheWohnenPlanungsursaechlich;
+
+    @Enumerated(EnumType.STRING)
+    @Column(nullable = true)
+    private SobonOrientierungswertJahr soBoNOrientierungswertJahr;
+
+    @Column(nullable = true)
+    private String anmerkung;
+
+    @OneToMany(cascade = { CascadeType.ALL }, fetch = FetchType.EAGER, orphanRemoval = true)
+    @JoinColumn(name = "abfragevariante_id", referencedColumnName = "id")
+    private List<BedarfsmeldungFachreferate> bedarfsmeldungFachreferate;
```

Comment:
Gibt es eine Sortierreihenfolge?

Falls keine Reihenfolge durch ein Attribute vorgegeben ist, sollte die Reihenfolge für das Erstellungsdatum der Entität verwendet werden.

javax.persistence.OrderBy

@OrderBy("createdDateTime asc")

Durch die Join-Operationen über die Tabellen, entspricht die Auslesereihenfolge beim Join nicht der Reihenfolge in der Tabelle.

Reply:
angepasst mit Abstimmung von @lhmAschaenz